test/integration/targets/dcnm_vrf/tests/nxapi/attach.yaml

```diff
@@ -0,0 +1,60 @@
+---
+- debug: msg="START connection={{ ansible_connection }}/create.yaml"
+
+- name: Verify if fabric - vxlan-shrishail-green-field is deployed.
+  dcnm_rest:
+    method: GET
+    path: /rest/control/fabrics/vxlan-shrishail-green-field
```

Comment:
Please add a `# TBD` comment to the end of this line. I just want something to jump out when we do our final reviews.

Reply:
Oh yeah, thanks for reminding, I will do that for all of these files.